dspace/bin/start-handle-server

```diff
@@ -21,19 +21,23 @@ HANDLEDIR=`$BINDIR/dspace dsprop --property handle.dir`
 # If you want your handle server logs stored elsewhere, change this value
 LOGDIR=$DSPACEDIR/log
 
-# Get the JARs in $DSPACEDIR/jsp/WEB-INF/lib, separated by ':'
-JARS=`echo $DSPACEDIR/lib/*.jar | sed 's/ /\:/g'`
+# Get the JARs in $DSPACEDIR/lib
+JARS=`$BINDIR/dspace classpath`
 
 # Class path for DSpace will be:
 #   Any existing classpath
-#   The JARs (WEB-INF/lib/*.jar)
-#   The WEB-INF/classes directory
-FULLPATH=$CLASSPATH:$JARS:$DSPACEDIR/config
+#   The JARs (lib/*.jar)
+#   The configuration directory
+if [ "$CLASSPATH" = "" ]; then
+  FULLPATH=$JARS:$DSPACEDIR/config
+else
+  FULLPATH=$CLASSPATH:$JARS:$DSPACEDIR/config
```

Comment:
Just reading the code, I *think* we can drop the `/config` directory from both the `if` and `else`, as it's already included by `./dspace classpath`.

In fact, we might be able to drop this entire `if/else` and ONLY call `./dspace classpath`.  But, that might need more testing

Reply:
`config/`:  you're right -- we can *and should* drop the additional copy.

`if/else`:  I consider this a sort of anti-bugging.  Without the `if` we may end up with an empty leading path element if CLASSPATH is undefined, which is common enough.  It's unlikely that a JRE would ever have serious trouble with that, but it's code out of our control....

Reply:
Actually `bin/dspace classpath` is already doing everything we need, so I just got rid of $JARS and the whole path-construction section.